pkg/receive/multitsdb.go

```diff
@@ -139,6 +136,60 @@ func NewMultiTSDB(
 	return mt
 }
 
+// getTenant returns the tenant with the given tenantID.
+// @testing: This method is only for testing purposes.
```

Comment:
Could we just add a `test` prefix as follows `testGetTenant` and remove this line? It will make it clearer what this method is for. People typically don't read comments :sweat_smile:

Reply:
fair points